### DISCUSSION TITLE
SSL stream isn't played

### DISCUSSION THREAD

**Discussion by JoergZ2** [MusicPlayerDaemon/MPD#1883] [Q&A]
Seems that german public stations (ARD) have change all streams to https protocol. Seems that MPD can't play such streams, isn't it? Or is there a trick to play? Address example: https://st01.sslstream.dlf.de/dlf/01/128/mp3/stream.mp3?aggregator=web (for browser use) or https://st01.sslstream.dlf.de/dlf/01/128/mp3/stream.mp3.

**Comment by leigh123linux** [comment]
I can play both URL using cantata mpd client.

**Comment by MaxKellermann** [comment]
https://www.musicpd.org/help/

**Reply by JoergZ2** [reply]
In the meantime I got the message that servers weren't available in the morning.
